File: frontend/src/app/dtos/tickets/ticket.dto.ts
export class TicketDto {
    constructor(
        public readonly startingTime: Date,
        public readonly eventName: string,
        public readonly locationName: string,
        public readonly roomName: string,
        public readonly price: number,
        public readonly secret: string,
        public readonly printableSeatDescription: string
    ) {}
}

File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/ticket/TicketDto.java
package at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.ticket;

import java.math.BigDecimal;
import java.sql.Timestamp;

public record TicketDto(
    Timestamp startingTime,
    String eventName,
    String locationName,
    String roomName,
    BigDecimal price,
    String secret,
    String printableSeatDescription) {}
